### PR TITLE
Rename generated fast profile to local profile

### LINKED ISSUE #602: Rename generated fast profile to local profile
Currently, in generator-jhipster, we have dev and prod profile, which is so confusing

In JHipster Lite, I created 'fast' profile but I don't think the name is correct. I think it's better to name it: local

### PR DESCRIPTION
**Pull request by qmonmert**
Fix #602

### CODE REVIEW

File: src/main/resources/generator/server/springboot/springcloud/configclient/src/bootstrap-local.properties.mustache [hunk 1]
```diff
@@ -11,4 +11,4 @@ spring.cloud.config.retry.initial-interval=1000
 spring.cloud.config.retry.max-attempts=100
 spring.cloud.config.retry.max-interval=2000
 spring.cloud.config.uri=http://admin:${jhipster.registry.password}@localhost:8761/config
-spring.cloud.config.fail-fast=false
+spring.cloud.config.fail-local=false
```
Comment (pascalgrimaud) [inline src/main/resources/generator/server/springboot/springcloud/configclient/src/bootstrap-local.properties.mustache:14]: Wrong change here

File: src/main/resources/generator/server/springboot/springcloud/configclient/src/bootstrap.properties.mustache [hunk 2]
```diff
@@ -11,4 +11,4 @@ spring.cloud.config.retry.initial-interval=1000
 spring.cloud.config.retry.max-attempts=100
 spring.cloud.config.retry.max-interval=2000
 spring.cloud.config.uri=http://admin:${jhipster.registry.password}@localhost:8761/config
-spring.cloud.config.fail-fast=true
+spring.cloud.config.fail-local=true
```
Comment (pascalgrimaud) [inline src/main/resources/generator/server/springboot/springcloud/configclient/src/bootstrap.properties.mustache:14]: Wrong change here

File: src/main/resources/generator/server/springboot/springcloud/consul/src/bootstrap-local.properties.mustache [hunk 3]
```diff
@@ -1,5 +1,5 @@
 #====================================================================
-# Spring Cloud Consul Config bootstrap configuration for fast profile
+# Spring Cloud Consul Config bootstrap configuration for local profile
 #====================================================================
 
-spring.cloud.consul.config.fail-fast=false
+spring.cloud.consul.config.fail-local=false
```
Comment (pascalgrimaud) [inline src/main/resources/generator/server/springboot/springcloud/consul/src/bootstrap-local.properties.mustache:5]: Wrong change

File: src/test/java/tech/jhipster/lite/generator/server/springboot/springcloud/configclient/application/SpringCloudConfigApplicationServiceIT.java [hunk 4]
```diff
@@ -103,15 +103,19 @@ private void assertProperties(Project project) {
...
-      getPath(MAIN_RESOURCES, "config/bootstrap-fast.properties"),
+      getPath(MAIN_RESOURCES, "config/bootstrap-local.properties"),
       bootstrapProperties(project.getBaseName().orElse("jhipster"))
     );
 
-    assertFileContent(project, getPath(MAIN_RESOURCES, "config/bootstrap-fast.properties"), List.of("spring.cloud.config.fail-fast=false"));
+    assertFileContent(
+      project,
+      getPath(MAIN_RESOURCES, "config/bootstrap-local.properties"),
+      List.of("spring.cloud.config.fail-local=false")
```
Comment (pascalgrimaud) [inline src/test/java/tech/jhipster/lite/generator/server/springboot/springcloud/configclient/application/SpringCloudConfigApplicationServiceIT.java:117]: Wrong change

**Review by pascalgrimaud — CHANGES_REQUESTED** [review]

File: src/main/resources/generator/server/springboot/springcloud/configclient/src/bootstrap-local.properties.mustache (hunk 1 above)
Comment (qmonmert) [reply src/main/resources/generator/server/springboot/springcloud/configclient/src/bootstrap-local.properties.mustache]: yes, fixed

File: src/main/resources/generator/server/springboot/springcloud/configclient/src/bootstrap.properties.mustache (hunk 2 above)
Comment (qmonmert) [reply src/main/resources/generator/server/springboot/springcloud/configclient/src/bootstrap.properties.mustache]: yes, fixed

File: src/test/java/tech/jhipster/lite/generator/server/springboot/springcloud/configclient/application/SpringCloudConfigApplicationServiceIT.java (hunk 4 above)
Comment (qmonmert) [reply src/test/java/tech/jhipster/lite/generator/server/springboot/springcloud/configclient/application/SpringCloudConfigApplicationServiceIT.java]: yes, fixed

File: src/main/resources/generator/server/springboot/springcloud/consul/src/bootstrap-local.properties.mustache (hunk 3 above)
Comment (qmonmert) [reply src/main/resources/generator/server/springboot/springcloud/consul/src/bootstrap-local.properties.mustache:5]: yes, fixed

**Review by pascalgrimaud — APPROVED** [review]

**Comment by pascalgrimaud** [comment]
@qmonmert : maybe the description should be fix xxx to auto close the ticket ?

**Pull request merged**